src/vorta/views/misc_tab.py

```diff
@@ -31,6 +35,10 @@ def __init__(self, parent=None):
         self.checkboxLayout.setContentsMargins(0, 0, 0, 12)
         self.checkboxLayout.setFormAlignment(Qt.AlignmentFlag.AlignHCenter)
 
+        # connect to settings model update
+        # the decorator doesn't work on methods
```

Comment:
Hi! I've tested this on my local machine and it works like a charm, I was trying to figure this out by using decorators but saw here about the decorators! (my approach was wrong 😅)

Reply:
Struggled with that one too, until I realized it wouldn't work on methods.